lib/hider/accounts/user.ex

```diff
@@ -73,4 +75,15 @@ defmodule Hider.Accounts.User do
       validate_required(changeset, :password)
     end
   end
+
+  def put_encrypt_cpf(changeset) do
+    cpf = get_field(changeset, :cpf)
+
+    cpf_encrypted = AES.encrypt(cpf)
```

Comment:
Generate the encrypted data (meaning that can be decrypted) thus, more critical, to it's `IV/salted`